statistics/handle/handle_test.go

```diff
@@ -2091,3 +2091,13 @@ func (s *testStatsSuite) TestExtendedStatsPartitionTable(c *C) {
 	err = tk.ExecToErr("alter table t2 add stats_extended s1 correlation(b,c)")
 	c.Assert(err.Error(), Equals, "Extended statistics on partitioned tables are not supported now")
 }
+
+func (s *testStatsSuite) TestHideIndexUsageSyncLease(c *C) {
+	// NOTICE: remove this test when index usage is GA.
+	defer cleanEnv(c, s.store, s.do)
+	tk := testkit.NewTestKit(c, s.store)
+	rs := tk.MustQuery("select @@tidb_config").Rows()
```

Comment:
could you also add a test about `show config`?

Reply:
Updated.